index.html

```diff
@@ -659,6 +659,7 @@ <h3>Presentational Children</h3>
 				</dd>
 			</dl>
 			<p>The <abbr title="Document Object Model">DOM</abbr> descendants are presentational. [=user agents=] SHOULD NOT expose descendants of this <a>element</a> through the platform <a>accessibility <abbr title="Application Programing Interface">API</abbr></a>. If [=user agents=] do not hide the descendant nodes, some information may be read twice.</p>
+			<p>Authors MUST NOT specify <pref>aria-owns</pref> on an element which has Presentational Children</p>
```

Comment:
```suggestion
			<p>Authors MUST NOT specify <pref>aria-owns</pref> on an element which has <a href="#childrenArePresentational">Presentational Children</a>.</p>
```

Reply:
I don't think it is necessary to link here as this is within the presentational children section

Reply:
ah, good point.  just the period then.